lib/linguist/languages.yml

```diff
@@ -4883,6 +4883,14 @@ XML:
   - Web.config
   - packages.config
   language_id: 399
+XPM:
+  type: data
+  extensions:
+  - ".xpm"
+  - ".pm"
+  ace_mode: c_cpp
+  tm_scope: none
```

Comment:
To treat this as C during syntax highlighting, you should write `source.c` here. You can use [Lightshow](https://github-lightshow.herokuapp.com/) to check that it works as expected.